### DISCUSSION TITLE
Other programming languages support

### DISCUSSION THREAD

**Discussion by murich** [Vanco/SequencePlugin#177] [Ideas & features]
Please make a guide on how to add support of other languages. 
I would like to add Solidity. 
Thank you.

**Comment by Vanco** [comment]
You can read source code for reference.    
Tips:   
Interface `IGenerator` for generate sequence call stack (a tree style text), witch will render as diagram.
Interface `SequenceNavigable` for navigation code when click diagram.  

You can give a try.
